numpy/_core/src/multiarray/stringdtype/casts.cpp

```diff
@@ -589,27 +790,9 @@ string_to_pylong(char *in, int has_null,
     return pylong_value;
 }
 
+template<typename TNpyLongType>
```

Comment:
can you call this `ToNpyLongType` maybe? Unless the T means something different. I find this name a little inscrutable.

Reply:
How about `NpyLongType`?